test/helpers/mock-lazyload.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+jest.mock('react-lazyload', () => ({
+    __esModule: true,
+    default: ({children}: React.PropsWithChildren<object>) => {
+        return (
+            <div>{children}</div>
```

Comment:
Not sure why this would need the div wrapper, maybe the lazy load implementation returns one too?